tools/rccl_replayer/rcclReplayer.cpp

```diff
@@ -137,14 +146,30 @@ void PrintGroupCall(GroupCall const& gc)
     for (int task = 0; task < rd.second.tasks.size(); task++) {
       TaskInfo ti = rd.second.tasks[task];
       std::string funcName = (ti.funcType == ncclCollSend || ti.funcType == ncclCollRecv) ? "Send/Recv" : ncclFuncNames[ti.funcType];
-      std::tuple<std::string, size_t, int, int> key(funcName, ti.count, ti.datatype, ti.op);
       printf("  - Task %02d: %32s inPlace=%d count=%lu datatype=%d op=%d root=%d\n",
              task, funcName.c_str(), ti.inPlace, ti.count, ti.datatype, ti.op, ti.root);
     }
   }
 }
 
 
+void dataToCsv(GroupCall const& gc, std::ofstream &datafile, double runTime)
+{
+  auto rd = *(gc.rankData.begin());
+  TaskInfo ti = rd.second.tasks[0];
+  std::string funcName = (ti.funcType == ncclCollSend || ti.funcType == ncclCollRecv) ? "Send/Recv" : ncclFuncNames[ti.funcType];
+  double n = (double) (ti.count);
+  double S = (double) (n * (double)DataTypeToBytes(ti.datatype));
+  double t = (double) (runTime/1000); //milliseconds to seconds
+  double busBw = (S/t);
+  if (funcName == "AllReduce") busBw *= (2*(n- 1)/n);
+  else if (funcName == "ReduceScatter" || funcName == "AllGather") busBw *= ((n-1)/n);
+  busBw /= (1024 * 1024 * 1024); //in gb/s
```

Comment:
To be consistent with rccl-tests, we should use GB/s, not GiB/s:
(1e9 vs 1024^3):
https://github.com/ROCm/rccl-tests/blob/c5cae38bb87c85ca1386e3c6f52cd1b03959ef37/src/all_reduce.cu#L37